From ebb3455b4f1e6362688b638d7805a64f3c3d1eeb Mon Sep 17 00:00:00 2001
From: Iain Hibbert <plunky@netbsd.org>
Date: Sun, 23 Feb 2020 21:39:25 +0000
Subject: [PATCH 2/6] remove extraneous semicolon

this semicolon causes a test to fail with read error, using g95
---
 TESTING/dbal.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TESTING/dbal.in b/TESTING/dbal.in
index c00fe3e83..828629b15 100644
--- a/TESTING/dbal.in
+++ b/TESTING/dbal.in
@@ -209,6 +209,6 @@ DBL:  Tests DGEBAL
 0.00000000000000000000D+00 0.00000000000000000000D+00 0.00000000000000000000D+00 0.63043209914231165391D-03 0.10000000000000000000D+01 0.79310682416114036641D+03
 0.00000000000000000000D+00 0.00000000000000000000D+00 0.00000000000000000000D+00 0.00000000000000000000D+00 0.12608641982846233078D-02 0.10000000000000000000D+01
 
-  2.494800386918399765D+291  1.582914569427869018D+175  1.004336277661868922D+59  3.186183822264904554D-58  5.053968264940243633D-175  0.40083367200179455560D-291;
+  2.494800386918399765D+291  1.582914569427869018D+175  1.004336277661868922D+59  3.186183822264904554D-58  5.053968264940243633D-175  0.40083367200179455560D-291
 
 0
